lib/app.tsx

```diff
@@ -258,10 +262,6 @@ export const App = connect(
         return window.print();
       }
 
-      if ('focusSearchField' === command.action) {
-        return this.props.focusSearchField();
-      }
-
```

Comment:
why is the import added above and then this block removed from here?

Reply:
it doesn't need to be special-cased if it's explicitly imported. I think having the special case here block was a mistake

Reply:
(also, sorry, this is a fix unrelated to the dialogs, I just saw it in passing 🤷‍♀ )

Reply:
> I think having the special case block here was a mistake. (The following lines check to see if it's a callable action and then call it.)

ah that makes sense. it _wasn't_ a mistake - I was rebelling against the next few lines by trying to make it explicit what actions we intend on responding to so we can eventually get rid of that `canCall` thing. too many headaches trying to understand the control flow of actions from Electron

Reply:
Oh!!! Okay, should I revert it? Right now since it's the only special-cased one it seemed wrong.

Reply:
Reverted!